cli.js

```diff
@@ -54,6 +68,30 @@ const formatOptionValidator = (opt) => {
 	return option;
 };
 
+/**
+ * Validator method for the size option flag
+ * @param opt {string} The size option flag value
+ * @returns {string} The processed size option flag value
+ */
+const sizeOptionValidator = (opt) => {
+
+	const allowedSizes = Object.entries(SUPPORTED_SIZES).map(([size, allowed]) => {
+		if (allowed) return size;
+	}).concat('all');
```

Comment:
Which meaning has the 'all' option here ? The _format_ option can actually contain multiple entries (leading to multiple outputs) with 'all' as a default option. Do we wan't to allow multiple output sizes in one run ? And if so, will your change produce multiple outputs in different formats ?

It looks to me, that you actually took the _formatOptionValidator_ and copy-pasted the function, but the logic here shall be a little different i think: checking if only one valid output size was chosen by the user.